desktop/renderer-app/src/apiMiddleware/Rtc.ts

```diff
@@ -71,6 +70,10 @@ export class Rtc {
             }
         }
 
+        this.rtcEngine.enableVideo();
+        // prevent camera being turned on temporarily right after joining room
+        this.rtcEngine.enableLocalVideo(false);
```

Comment:
does `enableLocalVideo` trigger early better:

```suggestion
        // prevent camera being turned on temporarily right after joining room
        this.rtcEngine.enableLocalVideo(false);
        this.rtcEngine.enableVideo();
```

I'm not sure, because I personally feel that the previous practice, may result in: first open the camera, and then close the camera

Reply:
Both `enableLocalVideo` and `enableVideo` will not take any effect until the `joinChannel` is called. And since `enableVideo` resets the whole camera states which includes the `enableLocalVideo`, putting `enableLocalVideo` after `enableVideo` should be safer.

Reply:
ok